piker/ui/_style.py

```diff
@@ -49,15 +51,27 @@ class DpiAwareFont:
 
     def __init__(
         self,
-        # TODO: move to config
         name: str = 'Hack',
-        font_size: str = 'default',
+        _font_size_key: str = 'default',
 
     ) -> None:
+
+        self._font_size_calc_key: str = _font_size_key
+        self._font_size: int | None = None
```

Comment:
this now gets cached (after first calc) if **not** defined in the user config `conf.toml`

TODO: ideally in the future we can actually build an entire "dynamic resizing system" + controls so that a user can do the classic `ctl` and `+/-` to change the font sizing (and thus all related graphics) interactively from the GUI. Not sure yet how difficult that's going to be to implement but in theory not that bad since pretty much all widgets have resize handlers.

Reply:
Hmm maybe i'll make a follow up task-issue for this?

Reply:
Oh yeah, another thing was checking out that https://github.com/samuelcolvin/watchfiles lib to see if we could also get a "changes-in-`conf.toml`-triggers-live-update" type thing (like `alacritty`) workin 😎

Reply:
Mentioned in #221 since we're more or less officially exposing a top level config now.